test/02_permutation/CMakeLists.txt

```diff
@@ -23,16 +23,19 @@
  # THE SOFTWARE.
  #
  ###############################################################################
- set(PermutationCommonSources ${HIPTENSOR_COMMON_TEST_SOURCES}
- ${CMAKE_CURRENT_SOURCE_DIR}/permutation_resource.cpp
- ${CMAKE_CURRENT_SOURCE_DIR}/permutation_test.cpp)
+ set(PermutationCommonSources
+     ${HIPTENSOR_COMMON_TEST_SOURCES}
+     ${CMAKE_CURRENT_SOURCE_DIR}/permutation_resource.cpp
+     ${CMAKE_CURRENT_SOURCE_DIR}/permutation_test.cpp
+     )
 
 #  tests
-set (PermutationTestSources ${PermutationCommonSources}
-${CMAKE_CURRENT_SOURCE_DIR}/permutation_column_major_test.cpp
```

Comment:
Are we removing the col_major test? Maybe we should keep it for regression purposes?